test/algconvergence/partitioned_methods_tests.jl

```diff
@@ -158,6 +158,8 @@ sim = test_convergence(dts, prob_big, ERKN5(), dense_errors = true)
 @test sim.𝒪est[:L2]≈4 rtol=1e-1
 
 # Adaptive methods regression test
+sol = solve(prob, DPRKN5())
+@test length(sol.u) < 160
```

Comment:
That seems a bit high. Are you sure the adaptivity parameters are correct?

Reply:
One of the adaptive parameters is wrong, just fixed!